tests/greenheart/test_hopp_for_h2.py.skip

```diff
@@ -73,16 +73,16 @@ class TestHOPPForH2:
         sample_site['site_num'] = 1
         Site = SiteInfo(sample_site, solar_resource_file=solar_resource_file, wind_resource_file=wind_resource_file)
 
-        hybrid_plant, combined_pv_wind_power_production_hopp, combined_pv_wind_curtailment_hopp, \
+        hybrid_plant, combined_hybrid_power_production_hopp, combined_hybrid_curtailment_hopp, \
             energy_shortfall_hopp, annual_energies, wind_plus_solar_npv, npvs, lcoe, lcoe_nom = hopp_for_h2(
                 Site, scenario, technologies, wind_size_mw, solar_size_mw, storage_size_mw, storage_size_mwh,
                 storage_hours, wind_cost_kw, solar_cost_kw, storage_cost_kw, storage_cost_kwh, kw_continuous,
                 load, custom_powercurve, interconnection_size_mw, grid_connected_hopp=True
             )
 
         df_produced = pd.DataFrame()
-        df_produced['combined_pv_wind_power_production_hopp'] = [sum(combined_pv_wind_power_production_hopp)]
-        df_produced['combined_pv_wind_curtailment_hopp'] = [sum(combined_pv_wind_curtailment_hopp)]
+        df_produced['combined_hybrid_power_production_hopp'] = [sum(combined_hybrid_power_production_hopp)]
```

Comment:
Is there a reason this test module is disabled?

Reply:
We're moving away from the hopp_for_h2 code, so this code should be deleted soon.